### PR TITLE
font added plus a few styling changes

### PR DESCRIPTION
**Pull request by Mariam-mohamed110**
- a new variables file was created to be able to use certain variables globally (especially the font)
- buttons didnt used to work if selected (not sure if it was just my device)
- best to use % for img weight/ height as it makes the logo look less elongated

### CODE REVIEW

File: web/src/components/Header.js [hunk 1]
```diff
@@ -16,7 +16,9 @@ export default function Header() {
       <header>
         <img src={explainailogo} alt="logo" />
         <div>
-          <p>Like I'm</p>
+          <p>
+            <nobr>Like I'm </nobr>
```
Comment (mdapim) [inline web/src/components/Header.js:20]: I didn't want to change this initially came up as a linter error but would it be better to use a hex code for this @DomVinyard

**Review by mdapim — APPROVED** [review]

File: web/src/styles/variables.scss [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  appearance: none;
+  background-image: url("data:image/svg+xml;utf8,<svg fill='black' height='24' viewBox='0 0 18 24' width='24' xmlns='http://www.w3.org/2000/svg'><path d='M7 10l5 5 5-5z'/><path d='M0 0h24v24H0z' fill='none'/></svg>");
+  background-repeat: no-repeat, repeat;
+  background-position: right 1px top 50%, 0 0;
+  background-size: 15px auto, 50%;
+
+  @media (min-width: $phone-width) {
+    display: none;
+  }
+}
```
Comment (yuna-iwata) [inline web/src/styles/variables.scss:38]: looks much cleaner with separate variables file nice 👍

File: web/src/components/Header.scss [hunk 3]
```diff
@@ -56,13 +25,19 @@ header {
...
+      height: 30px;
+      font-weight: bold;
+      font-family: $fontStyle;
+      font-size: 19px;
       background-color: $bg-button-inactive;
       border: none;
       color: $btn-text-inactive;
       font-weight: bolder;
+      font-family: $fontStyle;
+
```
Comment (yuna-iwata) [inline web/src/components/Header.scss:40]: cute buttons :)

File: web/src/components/Header.js (hunk 1 above)
Comment (Ademsk1) [reply web/src/components/Header.js:20]: Better to just use 
```suggestion
            {/* eslint-disable-next-line react/no-unescaped-entities */}
          <p>Like I'm</p>
```

File: web/src/components/Header.js [hunk 4]
```diff
@@ -37,7 +39,7 @@ export default function Header() {
           </button>
           <select
             onChange={(e) => setAge(e.target.value)}
-            className={age === "Adult" ? "select-adult" : "select-pg"}
+            className="select-pg"
```
Comment (s-dilip) [inline web/src/components/Header.js:42]: Nice styling to get rid of the conditional render. The new style works well on mobile as well as desktop. Ready to merge! 👍🏾

**Review by Ademsk1 — COMMENTED** [review]
Works fine for me, doesn't fail any tests, this is ready to merge

File: web/src/components/Header.js (hunk 1 above)
Comment (DomVinyard) [reply web/src/components/Header.js:20]: Perhaps a more readable solution is to define this as a variable

```
const formLabel = "Like I'm"
```

and then include it in the file as a variable

```
<p>{formLabel}</p>
```

Comment (DomVinyard) [reply web/src/components/Header.js:20]: This also makes it easier to update in future

**Pull request merged**